awscli/customizations/cloudformation/yamlhelper.py

```diff
@@ -62,21 +63,19 @@ def _dict_representer(dumper, data):
     return dumper.represent_dict(data.items())
 
 
-def _add_yaml_1_1_boolean_resolvers(resolver_cls):
-    # CloudFormation treats unquoted values that are YAML 1.1 native
-    # booleans as booleans, rather than strings. In YAML 1.2, the only
-    # boolean values are "true" and "false" so values such as "yes" and "no"
-    # when loaded as strings are not quoted when dumped. This logic ensures
-    # that we dump these values with quotes so that CloudFormation treats
-    # these values as strings and not booleans.
-    boolean_regex = re.compile(
-        '^(?:yes|Yes|YES|no|No|NO'
-        '|true|True|TRUE|false|False|FALSE'
-        '|on|On|ON|off|Off|OFF)$'
-    )
-    boolean_first_chars = list(u'yYnNtTfFoO')
-    resolver_cls.add_implicit_resolver(
-        'tag:yaml.org,2002:bool', boolean_regex, boolean_first_chars)
+# CloudFormation treats unquoted values that are YAML 1.1 native
```

Comment:
Is there a particular reason we removed this function? I'd assume we would have kept it and instead just pass the `Resolver` class that is dangled as a property to the `YAML` instance. If we switch it back, it would help reduce the number of lines we are changing and repeating the logic.

Reply:
I put it back, thanks.